python/ycm/client/command_request.py

```diff
@@ -69,9 +71,50 @@ def RunPostCommandActionsIfNeeded( self ):
         vimsupport.JumpToLocation( self._response[ 'filepath' ],
                                     self._response[ 'line_num' ],
                                     self._response[ 'column_num' ] )
-    elif 'message' in self._response:
-      vimsupport.EchoText( self._response['message'] )
+    elif self._is_fixit_command:
+      if not len( self._response[ 'fixits' ] ):
+        vimsupport.EchoText( "No fixits found for current line" )
+      else:
+        fixit = self._response[ 'fixits' ][ 0 ]
+
+        # We need to track the difference in length, but ensuring we apply fixes
+        # in ascending order of insertion point.
+        fixit[ 'chunks' ].sort( key = lambda chunk:  (
+          str(chunk[ 'range' ][ 'start' ][ 'line_num' ])
+          + ','
+          + str(chunk[ 'range' ][ 'start' ][ 'column_num' ])
+        ))
+
+        # Remember the line number we're processing. -ve line number means we
```

Comment:
it means "negative". But I totally see your point - i could have written this in english! :)